static/js/submit-warning.js

```diff
@@ -0,0 +1,53 @@
+document.addEventListener('DOMContentLoaded', function () {
+  let form = document.querySelector('.form-context');
+  let submitFormButton = document.getElementById('submitForm');
+  let viewableCheckbox = document.getElementById('shareOnAutSPACEs');
+  
+  // Check that event listener is working
+  // if (viewableCheckbox) {
+  //   viewableCheckbox.addEventListener('change', function () {
+  //     console.log('viewableCheckbox checked:', viewableCheckbox.checked);
+  //   });
+  // }
+  
+  if (submitFormButton) {
```

Comment:
Using the jQuery [`$('#submitForm')`](https://api.jquery.com/jQuery/) instead of `getElementById` has the benefit that it never returns null, so using it would mean you could drop these checks.

Reply:
Good to know. Changed to jQuery and removed the if's.